src/util.js

```diff
@@ -1,4 +1,14 @@
+const redis = require('redis');
+
+const redisUrl = process.env.REDIS_URL || '';
+const redisClient = redis.createClient(redisUrl);
+const closeRedisClient = callback => {
```

Comment:
It doesn't seem like a callback is actually ever being passed to this function. This argument seems unnecessary.

Reply:
Fair enough.